packages/ERTP/src/paymentLedger.js

```diff
@@ -480,6 +504,7 @@ export const preparePaymentLedger = (
       return issuer;
     },
     mintPayment(newAmount) {
+      // @ts-expect-error checked cast
       newAmount = coerce(newAmount);
```

Comment:
Help? What's the cast here? And why didn't we need this `expect-error` before this PR?

Reply:
Ah... I suppose this is a consequence of the main change here...

```diff
- * @returns {T & RemotableBrand<{}, T>}
+ * @returns {M & RemotableBrand<{}, M>}
```

I'm still not clear on what's being cast to what.

Reply:
the fix for `prepareExo` made the types flow so many `any` are now erroring. I think this suppression is due to some problem with `coerce`. I said "checked" because `coerce` is ensuring at runtime that the amount is the correct kind.